manager/main.py

```diff
@@ -69,6 +74,15 @@ def set_default_headers(response):  # pylint: disable=unused-variable
 
         return response
 
+    def get_x_rh_identity():
+        return request.headers.get('x-rh-identity')
+
+    # Apply global throttling for all routes
+    @app.app.before_request
+    @limiter.limit(f"{CFG.api_max_rps} per second", key_func=get_x_rh_identity)
```

Comment:
Could be rewritten to lambda
```suggestion
    @limiter.limit(f"{CFG.api_max_rps} per second", key_func=lambda: request.headers.get("x-rh-identity"))
```